iron-ajax.html

```diff
@@ -459,6 +459,9 @@
         console.error(error);
       }
 
+      error.status = request.status;
+      error.statusText = request.statusText;
+
       if (request === this.lastRequest) {
         this._setLastError({
           request: request,
```

Comment:
I think lastError is the place to add this info. It'd also be valuable to add `request.xhr.response` as well